android/shopify_buy_plugin/build.gradle

```diff
@@ -40,6 +40,7 @@ dependencies {
     testCompile 'junit:junit:4.12'
     testCompile 'org.mockito:mockito-core:1.10.19'
     testCompile 'org.json:json:20140107'
+    testCompile 'org.robolectric:robolectric:3.4.2'
```

Comment:
Have you considered to mock android resources?
https://developer.android.com/training/testing/unit-testing/local-unit-tests.html